From: "J. R. Okajima" <hooanon05@yahoo.co.jp>
Date: Mon, 1 Apr 2013 14:17:48 +0900
Subject: [05/83] aufs: tiny, update SF URLs
Origin: http://sourceforge.net/p/aufs/aufs3-standalone/ci/ace6a748c73f4836dc614ed64a9a6e64ac0e894c

Signed-off-by: J. R. Okajima <hooanon05@yahoo.co.jp>
(cherry picked from commit 886254ee19efb290e732352d82a3778f8e3e584d)
Backported from aufs3.4 since aufs3.2 is still alive.

Signed-off-by: J. R. Okajima <hooanon05@yahoo.co.jp>
---
 Documentation/filesystems/aufs/README | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Documentation/filesystems/aufs/README b/Documentation/filesystems/aufs/README
index 2137c15..c9f8097 100644
--- a/Documentation/filesystems/aufs/README
+++ b/Documentation/filesystems/aufs/README
@@ -115,20 +115,20 @@ represents the linux kernel version, "linux-3.x". For instance,
 
 o aufs3-linux tree
 $ git clone --reference /your/linux/git/tree \
-	git://aufs.git.sourceforge.net/gitroot/aufs/aufs3-linux.git \
+	git://git.code.sf.net/p/aufs/aufs3-linux aufs-aufs3-linux \
 	aufs3-linux.git
 - if you don't have linux GIT tree, then remove "--reference ..."
 $ cd aufs3-linux.git
 $ git checkout origin/aufs3.0
 
 o aufs3-standalone tree
-$ git clone git://aufs.git.sourceforge.net/gitroot/aufs/aufs3-standalone.git \
+$ git clone git://git.code.sf.net/p/aufs/aufs3-standalone \
 	aufs3-standalone.git
 $ cd aufs3-standalone.git
 $ git checkout origin/aufs3.0
 
 o aufs-util tree
-$ git clone git://aufs.git.sourceforge.net/gitroot/aufs/aufs-util.git \
+$ git clone git://git.code.sf.net/p/aufs/aufs-util \
 	aufs-util.git
 $ cd aufs-util.git
 $ git checkout origin/aufs3.0
